config/jobs/kubernetes-sigs/cluster-api/cluster-api-ci.yaml

```diff
@@ -28,6 +28,7 @@ periodics:
   labels:
     preset-dind-enabled: "true"
     preset-kind-volume-mounts: "true"
+  run_if_changed: '^((api|bootstrap|cmd|config|controllers|controlplane|errors|exp|feature|test|util)/|main\.go|go\.mod|go\.sum|Dockerfile|Makefile|scripts/ci-e2e\.sh)'
```

Comment:
Do we need to include `third_party`?

Reply:
+1 to this, we actually want to exclude docs and include everything else (might be also hack)

Reply:
Added third_party/, hack/ and whole scripts/.